core/controllers/dashboard_test.py

```diff
@@ -463,6 +463,76 @@ def test_no_explorations(self):
         self.assertEqual(response['explorations_list'], [])
         self.logout()
 
+    def test_no_explorations_and_visit_dashboard(self):
+        self.login(self.OWNER_EMAIL)
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(response['explorations_list'],[])
+
+        # Testing that creator only visit dashboard without any exploration created
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']),0)
+        
```

Comment:
Still this behaviour is being tested twice in this particular test. Can we please remove the duplication ?